CONTRIBUTING.md

```diff
@@ -166,13 +166,13 @@ like [pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv),
 Next, get an up to date checkout of the `drf-user` repository:
 
 ```sh
-$ git clone git@github.com:101loop/drf-user.git
+$ git clone git@github.com:<your-username>/drf-user.git
 ```
 
 or if you want to use git via `https`:
 
 ```sh
-$ git clone https://github.com/101loop/drf-user.git
+$ git clone https://github.com/<your-github-username>/browser-history.git
 ```
```

Comment:
@HardipinderS Actually, we have to add another section for using forked copy of this repo.
You can take reference from this link https://github.com/101Loop/APIManager-Flutter/blob/master/CONTRIBUTING.md

Reply:
Added changes I've pushed hopefully it will get added

Reply:
Added the link directly wasn't sure how much I could add